website/source/docs/drivers/docker.html.md

```diff
@@ -278,6 +283,22 @@ task "example" {
 }
 ```
 
+Example docker-config, using two helper scripts in $PATH,
+"docker-credential-ecr" and "docker-credential-vault":
+
+```json
+{
+  "auths": {
+    "internal.repo": { "auth": "`echo -n '<username>:<password>' | base64 -w0`" }
+  },
+  "credHelpers": {
+      "<XYZ>.dkr.ecr.<region>.amazonaws.com": "vault-ecr-login"
+  },
+  "credsStore": "vault"
```

Comment:
Sure thing, but ideally, maybe we should even (after this PR) write a small example-script to pull credentials from vault? I suspect more people would find that useful, since secretservice is more oriented to desktop, I believe.